### PR TITLE
6th section of community page w/signoff

### LINKED ISSUE #1271: New page design implementation for Contributors page
#### Current State of the Meshery.io site

Currently the page https://meshery.io/community/ is empty

#### Desired State of the Meshery.io site

Need to implement the mockup for this page from [Figma design](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=16435%3A39494&mode=design&t=6qls5jOGn5gJKKgm-1)
Contributors can pick separate section to implement the page fast

- [ ] Section-1: Banner section

image

- [ ] Section- 2: Resources section

image

- [ ] Section-3: Handbook section

image

- [ ] Section-4: Discuss forum section

image

- [ ] Section-5: Meshmate section

image

- [ ] Section-6: Calendar section"

image

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by ayushsinha25**
**Description**

This PR fixes #1271 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: community.html [hunk 1]
```diff
@@ -4,7 +4,72 @@
...
+      
+    </div>
+    <div class="text " style="width: auto; ">
+        <h1 style="font-family: 'Qanelas Soft', sans-serif; font-size: 3vmax; margin-bottom: 20px; margin-top: 20px;">Meshery Community <br>Meetings and Events</h1>
+        <p style="text-align: justify; margin-bottom: 20px; font-weight: 100; font-size: 14px;" >
+          Join Meshery’s meeting from anywhere in 
+        the world! We you missed on any of our events we 
+        have it recorded on our Youtube channel. You can also 
+        subscribe to our community calendar to stay up-to-date 
+        on all meetings and events.
```
Comment (thisiskaransgit) [inline community.html:28]:
```suggestion
          Join Meshery’s meeting from anywhere in the world! We you missed any of our events we have it recorded on our Youtube channel. You can also subscribe to our community calendar to stay up-to-date on all meetings and events.
```

File: community.html [hunk 2]
```diff
@@ -4,7 +4,72 @@
...
+        </p>
+        
+        <div class="logoss" style="display: flex;align-items: center;justify-content: left;">
```
Comment (thisiskaransgit) [inline community.html:31]:
```suggestion
        <div class="logos" style="display: flex;align-items: center;justify-content: left;">

```
- Consider renaming the classes, please. 
- Also, look for the globally available elements

File: assets/images/logos/YouTube-dark.svg
Comment (thisiskaransgit) [inline assets/images/logos/YouTube-dark.svg:1]: I am pretty sure the youtube logo (both the dark and light) looks chipped off.

File: community.html [hunk 3]
```diff
@@ -4,7 +4,72 @@
...
+          data-logo-for-dark="/assets/images/logos/Calendar Dark Theme.svg"
+          data-logo-for-light="/assets/images/logos/Calendar Light Theme.svg"
+          alt="Calendar Image"
+          id="logo-dark-light"
+        />
+       
+      
+    </div>
+    <div class="text " style="width: auto; ">
+        <h1 style="font-family: 'Qanelas Soft', sans-serif; font-size: 3vmax; margin-bottom: 20px; margin-top: 20px;">Meshery Community <br>Meetings and Events</h1>
```
Comment (thisiskaransgit) [inline community.html:22]: Keep the heading in the center when the container arrangement changes, for smaller screens.

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@ayushsinha25, thank your for your efforts

**Comment by leecalcote** [comment]
Keep up the good work, @ayushsinha25

**Comment by ayushsinha25** [comment]
> @ayushsinha25, thank your for your efforts 

Should I make all those changes that you have recommended? And a new pr? @thisiskaransgit

**Comment by sudhanshutech** [comment]
Lets consider this pr for merge i guess and raise another issue for enahancements // @thisiskaransgit @vishalvivekm

**Review by sudhanshutech — APPROVED** [review]

**Review by Rexford74 — APPROVED** [review]
You've put in a good shift, @ayushsinha25. Well done!

**Comment by ayushsinha25** [comment]
> You've put in a good shift, @ayushsinha25. Well done!

Thanks man!!!

**Comment by vishalvivekm** [comment]
@ayushsinha25 
We could not cover this in the last call, Please add it in today's websites call [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)
Let's discuss this today.

**Review by thisiskaransgit — APPROVED** [review]
We will be soon moving ahead with merging each section. Looks good to merge, thank you for your contribution 🎉

**Comment by Chadha93** [comment]
merge conflicts @ayushsinha25

**Comment by Ghat0tkach** [comment]
hii @ayushsinha25 

Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

Make sure to attend tommorow's website call for review

**Comment by ayushsinha25** [comment]
> hii @ayushsinha25 
> 
> Let's discuss it on the websites call.
> Please add this as an agenda item in the meeting minutes, if you would :)
> 
> https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima
> 
> Make sure to attend tommorow's website call for review 

Ya, sure

**Comment by vishalvivekm** [comment]
@ayushsinha25 please resolve the conflicts so that we can proceed with merging your efforts here.

**Comment by ayushsinha25** [comment]
> @ayushsinha25 please resolve the conflicts so that we can proceed with merging your efforts here.

Fine, will do it by Today evening

**Comment by ayushsinha25** [comment]
> @ayushsinha25 please resolve the conflicts so that we can proceed with merging your efforts here.

done

**Comment by thisiskaransgit** [comment]
Great job on your PR @ayushsinha25! Your efforts and contributions to the project are highly appreciated.
Just wanted to inform you that we will be closing this PR as another one with similar changes has been merged. Please know that this decision does not diminish the value of your work.
Keep up the fantastic work, and we look forward to seeing more of your contributions in the future!

**Pull request closed without merging**